proprietary/Components/src/denhaag/unordered-list.tokens.json

```diff
@@ -1,17 +1,39 @@
 {
   "denhaag": {
     "unordered-list": {
-      "color": { "value": "{denhaag.color.grey.4}" },
-      "marker-font-size": { "value": "1rem" },
-      "lower-alpha-list-style-type": { "value": "lower-alpha" },
-      "paragraph": {
-        "margin-block-start": { "value": "{denhaag.space.block.md}" }
+      "color": {
+        "value": "{denhaag.color.grey.4}"
       },
       "lead": {
-        "margin-block-start": { "value": "{denhaag.space.block.2xl}" }
+        "margin-block-start": {
+          "value": "{denhaag.space.block.2xl}"
+        }
+      },
+      "link-label": {
+        "hyphens": {
+          "value": "auto"
+        }
+      },
+      "link-with-icon": {
```

Comment:
Hier zou ik niet eens een CSS variabel maken. Dit is een variant op huidig variabel.

Nu gaan we een CSS variabel over een CSS variabel zetten, die waarschijnlijk al door een andere CSS variabel wordt gedefineerd :P